framework/bundles/org.jboss.tools.rsp.server/src/main/java/org/jboss/tools/rsp/server/model/internal/publishing/DeployableDelta.java

```diff
@@ -51,18 +69,22 @@ public void registerChange(FileWatcherEvent event) {
 		}
 	}
 	
-	public void clearDelta() {
+	/**
+	 * Clears all the changes that were registered in this delta.
+	 */
+	public void clear() {
 		changes.clear();
 	}
 	
 	private int convert(WatchEvent.Kind<?> kind) {
 		if( kind == StandardWatchEventKinds.ENTRY_CREATE)
 			return CREATED;
-		if( kind == StandardWatchEventKinds.ENTRY_MODIFY)
```

Comment:
These 'else' additions aren't strictly necessary, since if it enters the if, it returns from the function. So the old code was basically equivelant to an else-if without stating it ;)

Reply:
@robstryker sure, pure cosmetic. I consider adding the else contributes to better readability: state explicitly that it's an "else"-case vs discover it by spotting the "return" in between.